.circleci/config.yml

```diff
@@ -496,7 +496,7 @@ jobs:
                   - run:
                       name: Docker push
                       command: |
-                        docker push filecoin/<<parameters.image>>:<<parameters.channel>>-<<parameters.network>>
+                        docker push filecoin/<<parameters.image>>:<<parameters.channel>>
```

Comment:
I think this broke stuff?

Reply:
Ah, shoot. I did! Not sure how that change snuck in there, I only meant to do the other two. I'll open a new PR now.

Reply:
https://github.com/filecoin-project/lotus/pull/10096/files